src/browser_action/animate.css

```diff
@@ -0,0 +1,3272 @@
+@charset "UTF-8";
```

Comment:
I've been trying to put 3rd party js/css in it's own vendor folder. I'm happy to move this after merging as well, though.